### PR TITLE
[Stopwatch] Add name property to the stopwatchEvent

### LINKED ISSUE #14051: [Stopwatch] Add name property to the stopwatchEvent
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#37676
| PR author(s) | @AhmedRaafat14
| Merged in    | 5.2-dev

### PR DESCRIPTION
**Pull request by AhmedRaafat14**
Q | A
-- | --
Branch? | master for features
Bug fix? | no
New feature? | yes
Deprecations? | no
Tickets | Fix  symfony/symfony-docs#14051

### CODE REVIEW

File: components/stopwatch.rst [hunk 1]
```diff
@@ -102,6 +102,7 @@ For example::
     $event->getEndTime();    // returns the end time of the very last period
     $event->getDuration();   // returns the event duration, including all periods
     $event->getMemory();     // returns the max memory usage of all periods
+    $event->getName();      // returns the name of the event
```
Comment (noniagriconomie) [inline components/stopwatch.rst:105]:
```suggestion
    $event->getName();       // returns the name of the event
```

:)

**Review by noniagriconomie — APPROVED** [review]
you should probably as well update the PR desc to also mentioned `fix https://github.com/symfony/symfony-docs/issues/14051`
:)

**Comment by AhmedRaafat14** [comment]
@noniagriconomie  Updated thanks :)

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please add a versionadded directive 👍🏻

**Comment by javiereguiluz** [comment]
Ahmad, thanks and congrats on your first Symfony Docs contribution! It was a nice issue number too --> #14000

@OskarStark I added the versionadded directive while merging.

**Pull request merged**